frontend/src/views/Worksheets/WorksheetView.js

```diff
@@ -161,7 +161,7 @@ const WorksheetView = () => {
             page: 1,
             pageSize: 10000,
             term: '',
-            itemTypes: 'DatasetTable'
```

Comment:
If we remove this line, what happens with folders? Aren't we going to have response nodes for the folders? I think it would only affect the case in which we have a dataset of folders and no tables

Reply:
Good catch! Let me push a fix here